CONTRIBUTING.md

```diff
@@ -16,12 +16,31 @@ When migrating a component to master, or submitting a new component, please add
 
 ### Branch Structure
 
-All stable releases are tagged ([view tags](https://github.com/mui-org/material-ui/tags)). At any given time, `master` represents the latest development version of the library.
+All stable releases are tagged ([view tags](https://github.com/mui-org/material-ui/tags)).
+At any given time, `master` represents the latest development version of the library.
 Patches or hotfix releases are prepared on an independent branch.
 
 #### `master` is unsafe
 
-We will do our best to keep `master` in good shape, with tests passing at all times. But in order to move fast, we will make API changes that your application might not be compatible with.
+We will do our best to keep `master` in good shape, with tests passing at all times.
+But in order to move fast, we will make API changes that your application might not be compatible with.
+
+### How to increase the chance of being accepted?
+
+We will only accept a pull request that passes all our tests. Make sure the following is true:
+- The branch is not behind master.
+- If a feature was added:
+   - If this was already achievable with the core library explained why this
+      needs to be added to the core.
+   - Added tests.
+   - If this is a common use case considered adding an example to the documentation
+- If a bug was fixed added test cases that fail without the fix.
+- The code is formatted (run `yarn prettier`).
+- The code is linted (run `yarn lint`).
+- If documentation was changed in the source `yarn docs:api` was run.
+- If prop types were changed the TypeScript declarations were updated.
```

Comment:
```suggestion
- If prop types were changed, that the TypeScript declarations were updated.
```

Reply:
I would not write it that (great pun) way since the sentence reads weird in isolation. You always have to have the opening statement in mind for it to make sense.